### PR TITLE
test(aio): add tests for renderAttributes helper

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

File: aio/tools/transforms/helpers/utils.spec.js [hunk 1]
```diff
@@ -73,4 +73,27 @@ describe('utils', () => {
       );
     });
   });
+
+  describe('renderAttributes', () => {
+    it('should convert key-value map to a strong that can be used in HTML', () => {
```
Comment (gkalpak) [inline aio/tools/transforms/helpers/utils.spec.js]: _strong_ --> _string_

File: aio/tools/transforms/helpers/utils.spec.js [hunk 2]
```diff
@@ -73,4 +73,27 @@ describe('utils', () => {
...
+      expect(renderAttributes({ foo: 'bar', moo: 'car' })).toEqual(' foo="bar" moo="car"');
+    });
+
+    it('should handle boolean values', () => {
```
Comment (gkalpak) [inline aio/tools/transforms/helpers/utils.spec.js]: Only `true`, noot `false` :wink:

**Review by gkalpak — APPROVED** [review]

File: aio/tools/transforms/helpers/utils.spec.js (hunk 1 above)
Comment (petebacondarwin) [inline aio/tools/transforms/helpers/utils.spec.js]: +1

File: aio/tools/transforms/helpers/utils.spec.js (hunk 2 above)
Comment (petebacondarwin) [inline aio/tools/transforms/helpers/utils.spec.js]: The generator of the `attrMap` is not able to return a `false` value. If there is an attribute that says `a="false"` then the value in the map would be `"false"`.

But I guess someone else might provide such a value, in which case we would render `a="false"`.

**Comment by petebacondarwin** [comment]
@gkalpak PTAL

File: aio/tools/transforms/helpers/utils.spec.js (hunk 2 above)
Comment (gkalpak) [inline aio/tools/transforms/helpers/utils.spec.js]: I just meant the test description was not accurate, not that the implementation should support false values :grin:

**Review by gkalpak — APPROVED** [review]

**Pull request merged**